app/controllers/catalog_controller.rb

```diff
@@ -237,17 +237,17 @@ def show
     params[:id] = Druid.new(params[:id]).with_namespace
     _deprecated_response, @document = search_service.fetch(params[:id])
 
-    @cocina = Repository.find(params[:id])
+    @cocina = Repository.find_lite(params[:id], structural: false)
     flash[:alert] = "Warning: this object cannot currently be represented in the Cocina model." if @cocina.instance_of?(NilModel)
 
     authorize! :read, @cocina
 
     @workflows = WorkflowService.workflows_for(druid: params[:id])
-
     milestones = MilestoneService.milestones_for(druid: params[:id])
     object_client = Dor::Services::Client.object(params[:id])
+    versions = object_client.version.inventory
     @milestones_presenter = MilestonesPresenter.new(milestones:,
-      versions: object_client.version.inventory)
+      versions: versions)
```

Comment:
it's stupid to have this on a second line, IMO.

Reply:
Fixed.